### ISSUE TITLE
[PvP] Battlefields do not start - Opcode malfunction

### ISSUE THREAD

**Issue by Echinaceax** [WoWManiaUK/Blackwing-Lair#3341]
**Links:**

https://wow.gamepedia.com/Tol_Barad
https://www.youtube.com/watch?v=LKHYFCRtad4

![horde attacking](https://user-images.githubusercontent.com/39439201/65345404-7ee80880-dbda-11e9-99d7-00bea0eaa7e5.jpg)
![ally wins](https://user-images.githubusercontent.com/39439201/65345415-83142600-dbda-11e9-9026-8253e5be80cc.jpg)

**What is happening:**

- battle has problems starting 
but as soon as it starts you get teleported out like wintergrasp (WOTLK)
and alliance always wins, since nobody able to attack ^^
confirmed by Oorsnor is discord

Also i wasnt ported from dailies zone in Tol Barad ..
shouldnt be able to continue dailies when battle is on :)

Also 
Same issue as Wintergrasp #3368

**Comment by Rushor** [comment]
void WorldSession::HandleBfEntryInviteResponse(WorldPacket& recvData)

maybe sth is broke here

**Comment by Echinaceax** [comment]
yes ..even when the isnt enough players is shoulb do a "not enough players you will be port in 5mins" etc...surely ?

**Comment by Rushor** [comment]
okay, but the quests in the battlefields are doabe as solo player @Echinaceax ?

**Comment by Echinaceax** [comment]
> okay, but the quests in the battlefields are doabe as solo player @Echinaceax ?

yes correct @Rushor

**Comment by Rushor** [comment]
will combine those two https://github.com/WoWManiaUK/Blackwing-Lair/issues/3368

**What is happening:**
- Battle doesnt start 
- You get teleported out 
- No message for "not enough players" 
- get a blank join screen, when you click it, it doesnt join you to group  #1975

_Only message we get is "let the battle begin!" then teleported out_

Note: _Reported by players on live, and tested by me and Rusher on DEV_
